### PR TITLE
add PlaNus description and content page for user guide v1.2

### LINKED ISSUE #63: add PlaNus description and content page for user guide v1.2

### PR DESCRIPTION
**Pull request by MarcTzh**
fix #63

### CODE REVIEW

**Review by nuovonatura — APPROVED** [review]
LGTM.

File: docs/UserGuide.md [hunk 1]
```diff
@@ -1,48 +1,31 @@
...
-   * **`delete`**`3` : Deletes the 3rd contact shown in the current list.
-
-   * **`clear`** : Deletes all contacts.
-
-   * **`exit`** : Exits the app.
-
-1. Refer to the [Features](#features) below for details of each command.
+PlaNus is a desktop app for **managing tasks, optimized for use via a Command Line Interface (CLI)** while still having the benefits of a Graphical User Interface (GUI). If you can type fast, PlaNus can get your tasks managed faster than traditional GUI apps.
+
+* Quick start
```
Comment (BobbyZhouZijian) [inline docs/UserGuide.md:7]: quick start should start with ##

File: docs/UserGuide.md [hunk 2]
```diff
@@ -1,48 +1,31 @@
...
+* Features
```
Comment (BobbyZhouZijian) [inline docs/UserGuide.md:8]: The contents should be hyperlink

**Review by BobbyZhouZijian — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 2 above)
Comment (BobbyZhouZijian) [reply docs/UserGuide.md:8]: I will add the hyperlinks

**Review by BobbyZhouZijian — APPROVED** [review]
LGTM

**Pull request merged**